src/main/java/frc/robot/Constants.java

```diff
@@ -91,4 +91,7 @@
 		public static final int kDriverControllerPort = 0;
 		public static final double kControllerDeadband = 0.11;
 	}
+	public static final class ConversionConstants {
+		public static final short kDegreesInARotation = 360;
+	}
```

Comment:
constants are for storing numbers specific to our robot not the degrees in a circle